test/runtests.py

```diff
@@ -125,11 +126,16 @@
 if sys.platform != 'win32':
     not_tags.add('exclude_xplat')
     not_tags.add('Intl')
+    not_tags.add('require_simd')
+    not_tags.add('require_asmjs')
```

Comment:
Note: in the new version of the PR, this is changed so the require_asmjs flag is no longer there, since those tests now work on xplat.